obspy/io/nordic/core.py

```diff
@@ -1684,6 +1755,46 @@ def _write_header_line(event, origin, evtype, is_preferred_origin=True):
     return ''.join([''.join(line) for line in lines])
 
 
+def _write_high_accuracy_origin(origin):
+    """
+    Write high accuracy hypocenter line
+    E.g.:
+    1996  6 3 2006 35.511  46.78711  153.72245   33.011  1.923
+    """
+    # Write the header info.
+    if origin.latitude is not None:
+        lat = '{0:.5f}'.format(origin.latitude)
+    else:
+        lat = ''
+    if origin.longitude is not None:
+        lon = '{0:.5f}'.format(origin.longitude)
+    else:
+        lon = ''
+    if origin.depth is not None:
+        depth = '{0:.3f}'.format(origin.depth / 1000.0)
+    else:
+        depth = ''
+    evtime = origin.time
+    if not evtime:
+        return
+
+    # Cope with differences in event uncertainty naming
+    if origin.quality and origin.quality['standard_error']:
+        timerms = '{0:.3f}'.format(origin.quality['standard_error'])
+    else:
+        timerms = '0.000'
+
+    ha_origin_line = (
+        " {0} {1}{2} {3}{4} {5}.{6} {7} {8} {9} {10} {11}H\n".format(
+            evtime.year, str(evtime.month).rjust(2), str(evtime.day).rjust(2),
+            str(evtime.hour).rjust(2), str(evtime.minute).rjust(2),
+            str(evtime.second).rjust(2), str(evtime.microsecond).ljust(3)[0:3],
+            lat.rjust(9), lon.rjust(10), depth.rjust(8), timerms.rjust(6),
+            " " * 19))
```

Comment:
We can use f-strings nowadays, which makes it easier to read, e.g.

```python
f"... {evtime_day:2d} ..."   # rjust is default, "<2d" for ljust
```

instead of..

```
" .... {2} ....".format(...., str(evtime.day).rjust(2), ...)
```

but thats just a comment, if you don't wanna touch it again, thats OK

Reply:
Converting to f-strings would be great and really improve readability of my cludge of code, but it might be worth having a different PR for it unless @flixha wants to include it in this one?

Reply:
I'm fine with either I guess

Reply:
Yes it would be good to change this; but as Calum suggests this would warrant an independent PR to change all of these format strings in the code, while here I just stayed consistent with the rest and copied relevant parts from other spots in the code..